ambry-network/src/main/java/com.github.ambry.network/Transmission.java

```diff
@@ -132,8 +132,7 @@ public SocketAddress getRemoteSocketAddress() {
   /**
    * Close the connection for the socket channel
    */
-  public abstract void close()
-      throws IOException;
```

Comment:
To our knowledge (atleast for now), we have only plain text transmission and ssl transmission. both of them doesn't throw IOException on close. Hence fixing the abstract class. We can work on fixing other methods later on a demand basis. Since this came in our way, no harm in fixing it.